### PR TITLE
[zh-cn]: create translation of Smoke Test Glossary

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/glossary/smoke_test/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+- “程序能否正常启动？”
+- “主控制按钮是否正常工作？”
+- “能否保存一个简单的空白测试用户账户？”
+
+如果这些基本功能测试失败，则没有必要在此时投入时间进行更详细的 QA 工作。
+
+## 参见
+
+- 维基百科上的 [冒烟测试（软件）](<https://zh.wikipedia.org/wiki/%E5%86%92%E7%83%9F%E6%B5%8B%E8%AF%95_(%E8%BD%AF%E4%BB%B6)>)
```
Comment (jasonren0403) [inline files/zh-cn/glossary/smoke_test/index.md:22]:
```suggestion
- 维基百科上的[冒烟测试（软件）](<https://zh.wikipedia.org/wiki/冒烟测试_(软件)>)
```

File: files/zh-cn/glossary/smoke_test/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: 冒烟测试（Smoke Test）
+slug: Glossary/Smoke_Test
+l10n:
+  sourceCommit: ada5fa5ef15eadd44b549ecf906423b4a2092f34
+---
+
+{{GlossarySidebar}}
+
+冒烟测试是指对软件的关键功能进行的功能测试或单元测试。它是在进行更深入的测试之前进行的一种快速测试。
```
Comment (jasonren0403) [inline files/zh-cn/glossary/smoke_test/index.md:10]:
```suggestion
冒烟测试（Smoke Test）是指对软件的关键功能进行的功能测试或单元测试。它是在进行更深入的测试之前进行的一种快速测试。
```

File: files/zh-cn/glossary/smoke_test/index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: 冒烟测试（Smoke Test）
```
Comment (jasonren0403) [inline files/zh-cn/glossary/smoke_test/index.md:2]:
```suggestion
title: 冒烟测试
```

**Review by jasonren0403 — COMMENTED** [review]
nits

File: files/zh-cn/glossary/smoke_test/index.md (hunk 1 above)
Comment (T34-active) [reply files/zh-cn/glossary/smoke_test/index.md]: my bad😢

**Review by jasonren0403 — APPROVED** [review]
👍 LGTM

**Pull request merged**